functional-tests/make-binaries

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+go build -o ./sops go.mozilla.org/sops/cmd/sops
+BRANCH=`git rev-parse --abbrev-ref HEAD`
+git tag | grep "2.*" | xargs -I {} bash -c "git checkout {} && go build -o /tmp/sops{} go.mozilla.org/sops/cmd/sops"
```

Comment:
why only build `2.x` tags?

Reply:
Because we can't build 1.x, but yeah, building everything but 1.x is the right approach.